rpc-server/src/modules/network/methods.rs

```diff
@@ -325,70 +326,56 @@ async fn protocol_config_call(
                 }
             })?;
 
-    let protocol_config = if data
-        .blocks_info_by_finality
-        .final_cache_block()
-        .await
-        .epoch_id
-        == block.epoch_id
-    {
-        let store = near_parameters::RuntimeConfigStore::for_chain_id(
-            &data.genesis_info.genesis_config.chain_id,
-        );
-        let runtime_config = store.get_config(block.latest_protocol_version);
+    let store = near_parameters::RuntimeConfigStore::for_chain_id(
```

Comment:
Please, drop a comment explaining what is happening here if you've figured it out